.circleci/populate-wheelhouse.sh

```diff
@@ -30,5 +30,4 @@ LANG="en_US.UTF-8" "${PIP}" \
     wheel \
     --wheel-dir "${WHEELHOUSE_PATH}" \
     "${PROJECT_ROOT}"[testenv] \
-    "${PROJECT_ROOT}"[test] \
-    "${PROJECT_ROOT}"[build]
+    "${PROJECT_ROOT}"[test]
```

Comment:
This could help in [ticket:4133 (hatchling)](https://tahoe-lafs.org/trac/tahoe-lafs/ticket/4133) and/or [ticket:4143 (refresh images)](https://tahoe-lafs.org/trac/tahoe-lafs/ticket/4143), but has nothing to do with retiring Python 3.8 IMO.